styles/styles.css

```diff
@@ -34,6 +34,7 @@
   --breadcrumb-text-color-dark: #c5c5c5;
   --breadcrumb-text-color-dark-hover: #fff;
   --icon-color: #205fff;
+  --background-light-blue: #d9eeff;
```

Comment:
Should we name this var semantically to avoid renaming in case of a color change? Something like `--cta-section-background`?

Reply:
Would it make sense to blend all parts of the image and move it to the Sharepoint folder so that customer could easily replace it?

Reply:
> Should we name this var semantically to avoid renaming in case of a color change? Something like `--cta-section-background`?

Ok.

Reply:
> Would it make sense to blend all parts of the image and move it to the Sharepoint folder so that a customer could easily replace it?

For now, I think that only having the people image on doc is flexible enough.

Reply:
> flexible enough

I mean this way we could also remove both the image and an svg icon from the git repo :-)

Reply:
Created #87 to track this enhancement